BlazarBase/src/main/java/com/hubspot/blazar/base/ModuleState.java

```diff
@@ -11,46 +11,46 @@
 
   private final Module module;
   private final Optional<ModuleBuild> lastSuccessfulModuleBuild;
-  private final Optional<RepositoryBuild> lastSuccessfulRepoBuild;
+  private final Optional<RepositoryBuild> lastSuccessfulBranchBuild;
```

Comment:
Would it make sense now to have ModuleState contain ModuleBuildInfos instead of separate fields for module and branch (repo) builds?